components/hab/src/main.rs

```diff
@@ -498,6 +497,10 @@ fn exec_subcommand_if_called(ui: &mut UI) -> Result<()> {
         }
         ("sup", _) => command::sup::start(ui, env::args_os().skip(2).collect()),
         ("start", _) => command::sup::start(ui, env::args_os().skip(1).collect()),
+        ("service", "load") |
```

Comment:
It might make sense to group all Supervisor/Service-related verb-named subcommands under a `service` group, just like we do in `hab`. In other words, `hab-sup load` would be `hap-sup service load` (or similar). This would help to clarify exactly what `hap-sup service start` would do, as opposed to what `hab-sup start` (and by extension, `hab start`) might do.